### DISCUSSION TITLE
Program aborted "Error while executing test case"

### DISCUSSION THREAD

**Discussion by Realone24** [NASA-LIS/LISF#1034] [Q&A] [answered]
Dear all,

I have successfully compiled the LIS and all went well and it generated the exe. However, when I try to run it on the test case, it was showing me following errors.  My environment variables are listed below and I have also attached the configuration file.  It would be helpful if someone could provide me any suggestion, thanks.

export LIS_ARCH=linux_gfortran
export LIS_FC=/usr/bin/gfortran
export LIS_CC=/usr/bin/gcc
export LIS_MODESMF=/home/hydro/DATA_ASSIM/LIBS/esmf/mod/modO/Linux.gfortran.64.mpiuni.default
export LIS_LIBESMF=/home/hydro/DATA_ASSIM/LIBS/esmf/lib/libO/Linux.gfortran.64.mpiuni.default
export LIS_OPENJPEG=/home/hydro/DATA_ASSIM/LIBS/openjpeg
export LIS_ECCODES=/home/hydro/DATA_ASSIM/LIBS/ecCodes
export LIS_NETCDF=/home/hydro/DATA_ASSIM/LIBS/netcdf45

export LD_LIBRARY_PATH=$LIS_HDF5/lib:$LIS_LIBESMF:$LIS_NETCDF/lib:${LIS_ECCODES}/lib:$LD_LIBRARY_PATH

Program aborted. Backtrace:
#0  0x2B92F799CE08
#1  0x2B92F799DCE2
#2  0x2B92F7A78488
#3  0x588EF8 in __lis_logmod_MOD_lis_endrun
#4  0x5052B7 in __lis_domainmod_MOD_readdomaininput at LIS_domainMod.F90:?
#5  0x507D67 in __lis_domainmod_MOD_lis_domain_init
#6  0xD76FCD in __retrospective_runmod_MOD_lis_init_retrospective
#7  0x5C4F06 in lisinit_
Aborted (core dumped)

[configure.lis.txt](https://github.com/NASA-LIS/LISF/files/8389172/configure.lis.txt)

**Comment by bmcandr** [comment, marked as answer]
Hi @Realone24, thanks for your message. I have a few questions:

* What step in the public testcase were you running when the error occurred: 2 or 6? (the open loop or data assimilation step?)
* Also, can you attach a log file (e.g., lislog.0000)?

It may also be helpful if you re-compile lis with debug optimization so that a more informative stacktrace is produced. To do so change into the `lis/` directory, run `make -C make realclean`, run `./configure` and select -1 at prompt 2, then compile again.

**Reply by Realone24** [reply]
Hi Thanks for the reply, there were some files required to run the exe.
